src/models/Group.js

```diff
@@ -0,0 +1,14 @@
+export default function(sequelize, DataTypes) {
+    var Group = sequelize.define('Group', {
+        groupname: {
+            type: DataTypes.ENUM('TME', 'ADMIN', 'OUTLET'),
+        }
+    }, {
+        associate: (models) => {
+            Group.hasMany(models.Profile, { foreignKey: 'group_id' });
```

Comment:
again profile has group. group doesnt have profile.

user profile belongs to a group, not reverse